source/lex.tex

```diff
@@ -358,6 +416,10 @@
     named-universal-character
 \end{bnf}
 
+\pnum
+The \grammarterm{universal-character-name} construct provides a way to name
+any element in the translation character set using just the basic character set.
```

Comment:
That rephrasing is not correct: You can't use a UCN to name a basic character outside of a string-literal.

Reply:
I tested the following program to declare the identifier pi as a variable, and it was accepted by all 4 major front ends:
```
double \u03c0 = 3.14;
```

Does this compile for some other reason?  Or are all 4 implementations buggy in allowing UCNs outside of character and string literals?

https://godbolt.org/z/hnbWr9qeo

Reply:
I said "basic character". You're not naming one of the basic characters.
Try naming "A" with a UCN outside of a literal.

Reply:
In that case, I still think the text is good as written.  This is an introductory sentence, not a definition.  As far as I can tell it is still possible to name elements of the basic character set, but we limit where those particular names can be used to character and string literals --- it is not forming the name that fails, but how we use it after.  That all seems reasonable for the introductory paragraph.